From: SeongJae Park <sj@kernel.org>
Subject: Docs/admin-guide/mm/numaperf: change title to fix duplicate label warning
Date: Fri, 6 Jan 2023 19:49:27 +0000

Commit 1564fac284b1 ("Docs/admin-guide/mm/numaperf: increase depth of
subsections") in mm-unstable titled numaperf.rst as 'NUMA Performance'. 
The name overlaps with an existing subsection title of the document, so
sphinx makes below warning.

    linux/Documentation/admin-guide/mm/numaperf.rst:68: WARNING: duplicate label admin-guide/mm/numaperf:numa performance, other instance in linux/Documentation/admin-guide/mm/numaperf.rst

Fix the warning by changing the title of the document to 'NUMA Memory
Performance'.  It may look more consistent with its sibling document,
'NUMA Memory Policy'.

Link: https://lkml.kernel.org/r/20230106194927.152663-1-sj@kernel.org
Fixes: 1564fac284b1 ("Docs/admin-guide/mm/numaperf: increase depth of subsections") in mm-unstable
Signed-off-by: SeongJae Park <sj@kernel.org>
Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---


--- a/Documentation/admin-guide/mm/numaperf.rst~docs-admin-guide-mm-numaperf-increase-depth-of-subsections-fix
+++ a/Documentation/admin-guide/mm/numaperf.rst
@@ -1,8 +1,8 @@
 .. _numaperf:
 
-================
-NUMA Performance
-================
+=======================
+NUMA Memory Performance
+=======================
 
 NUMA Locality
 =============
_
